src/node/utils/cssUtils.ts

```diff
@@ -90,22 +91,6 @@ export async function compileCss(
     postcssOptions,
     postcssPlugins
   })
-
-  // record css import dependencies
-  if (res.rawResult) {
-    res.rawResult.messages.forEach((msg) => {
-      let { type, file, parent } = msg
-      if (type === 'dependency') {
-        if (cssImportMap.has(file)) {
-          cssImportMap.get(file)!.add(parent)
-        } else {
-          cssImportMap.set(file, new Set([parent]))
-        }
-      }
-    })
-  }
-
-  return res
```

Comment:
I moved this into each process for css and vue style.Because it's necessary for build.